etl/nysdoh_nursinghomes.py

```diff
@@ -0,0 +1,59 @@
+from dataflows import *
+from lib import dump_to_postgis, rename_field
+import os
+import csv
+import sys
+from utils import url, fields, geo_flow, get_the_geom, quick_clean, get_hnum, get_sname
+
+csv.field_size_limit(sys.maxsize)
+
+table_name = 'nysdoh_nursinghomes'
+nysdoh_nursinghomes = Flow(
+    load(url, resources = table_name, force_strings=False),
+    checkpoint(table_name),
+
+    filter_rows(equals = [
+            dict(state = 'NY')
+            ]),
+
+    filter_rows(equals = [
+            dict(county = 'New York'),
+            dict(county = 'Queens'),
+            dict(county = 'Bronx'),
+            dict(county = 'Richmond'),
+            dict(county = 'Kings'),
+            ]),
+
+    add_field('datasource', 'string', table_name),
+
+    ################## geospatial ###################
+    ###### Make sure the following columns ##########
+    ###### exist before geo_flows          ########## 
+    #################################################
+
+    rename_field('zip_code','zipcode'),
+    
+    add_computed_field([dict(target=dict(name = 'boro', type = 'string'),
```

Comment:
can you add a boro conversion function in utils? 
e.g. `convert_to_boro("New York") = 'MN'`, `convert_to_boro("Richmond") = 'SI'`
and include additional options such as `convert_to_boro(1) = 'MN'`,
``convert_to_boro("X") = 'BX'`
ideally every dataset would have the boro field, even if it has to be empty

Reply:
@SPTKL just updated it as you suggested.